webrender/src/device/gfx.rs

```diff
@@ -4772,7 +4929,9 @@ impl<B: hal::Backend> Device<B> {
                 .destroy_semaphore(self.image_available_semaphore);
             self.device
                 .destroy_semaphore(self.render_finished_semaphore);
-            self.device.destroy_swapchain(self.swap_chain.unwrap());
+            if self.swap_chain.is_some() {
```

Comment:
With `if let Some(swap_chain) = self.swap_chain` you don't need to check if it's a `Some(val)` or `None`.